Framework/CurveFitting/src/Functions/CubicSpline.cpp

```diff
@@ -231,7 +228,7 @@ void CubicSpline::calculateDerivative(double *out, const double *xValues, const
 
   // inform user that some values weren't calculated
   if (outOfRange) {
-    g_log.information() << "Some x values where out of range and will not be calculated.\n";
+    g_log.information() << "Some x values were out of range and will not be calculated.\n";
```

Comment:
nitpick
```suggestion
    g_log.information("Some x values were out of range and will not be calculated.");
```
will print the same thing with lower overhead